apps/passport-server/test/util/util.ts

```diff
@@ -63,7 +80,17 @@ export function expectLength<T>(
 
   if (array.length !== length) {
     throw new Error(
-      `expected ${JSON.stringify(array, null, 2)} to have length ${length}`
+      `expected ${JSON.stringify(
+        array,
+        function (_key, value) {
+          if (isTimeoutObject(value)) {
```

Comment:
I found this because I had a failing array length check that could not print out the affected array because it contained a timeout object. I couldn't find a better way around it than this.